.eslintrc.json

```diff
@@ -77,65 +99,52 @@
         // Pending https://github.com/typescript-eslint/typescript-eslint/issues/4820
         "@typescript-eslint/prefer-optional-chain": "off",
 
-        // Rules enabled in typescript-eslint configs that are not applicable here
-        "@typescript-eslint/ban-ts-comment": "off",
-        "@typescript-eslint/class-literal-property-style": "off",
-        "@typescript-eslint/consistent-indexed-object-style": "off",
-        "@typescript-eslint/no-duplicate-enum-values": "off",
-        "@typescript-eslint/no-namespace": "off",
-        "@typescript-eslint/no-non-null-asserted-optional-chain": "off",
-        "@typescript-eslint/no-var-requires": "off",
-
         // scripts/eslint/rules
-        "local/object-literal-surrounding-space": "error",
-        "local/no-type-assertion-whitespace": "error",
-        "local/type-operator-spacing": "error",
-        "local/only-arrow-functions": ["error", {
-            "allowNamedFunctions": true ,
-            "allowDeclarations": true
-        }],
-        "local/no-double-space": "error",
+        "local/only-arrow-functions": [
+            "error",
+            {
+                "allowNamedFunctions": true ,
+                "allowDeclarations": true
+            }
+        ],
         "local/argument-trivia": "error",
         "local/no-in-operator": "error",
-        "local/simple-indent": "error",
         "local/debug-assert": "error",
         "local/no-keywords": "error",
         "local/jsdoc-format": "error",
 
         // eslint-plugin-no-null
         "no-null/no-null": "error",
 
-        // eslint
+        // eslint-plugin-simple-import-sort
+        "simple-import-sort/imports": "error",
+        "simple-import-sort/exports": "error",
+
+        // Formatting rules; remove once a formatter enforces these.
         "curly": ["error", "multi-line"],
-        "dot-notation": "error",
-        "eqeqeq": "error",
         "linebreak-style": ["error", "windows"],
+        "max-statements-per-line": ["error", { "max": 1 }],
         "new-parens": "error",
-        "no-caller": "error",
-        "no-eval": "error",
-        "no-extra-bind": "error",
-        "no-new-func": "error",
-        "no-new-wrappers": "error",
-        "no-return-await": "error",
-        "no-restricted-globals": ["error",
-            { "name": "setTimeout" },
-            { "name": "clearTimeout" },
-            { "name": "setInterval" },
-            { "name": "clearInterval" },
-            { "name": "setImmediate" },
-            { "name": "clearImmediate" }
-        ],
-        "no-template-curly-in-string": "error",
-        "no-throw-literal": "error",
         "no-trailing-spaces": "error",
-        "no-undef-init": "error",
-        "no-var": "error",
-        "object-shorthand": "error",
-        "prefer-const": "error",
-        "prefer-object-spread": "error",
         "quote-props": ["error", "consistent-as-needed"],
         "space-in-parens": "error",
-        "unicode-bom": ["error", "never"]
```

Comment:
Snuck this in, but, this is just wholesale removed. I apparently missed this one in my review of Josh's PR as this one is already enabled by default in eslint.

Reply:
Ugh, I'm wrong; I misread the page and the default is "never", but it's not enabled by default, so I will restore this.